dali/dfu/dfuserver.cpp

```diff
@@ -149,9 +149,15 @@ int main(int argc, const char *argv[])
     if (!stop) {
         sentinelFile.setown(createSentinelTarget());
         removeSentinelFile(sentinelFile);
+        int maxdetail = 1000;
+#ifndef _CONTAINERIZED
         Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator(globals, "dfuserver");
-        lf->setMaxDetail(1000);
+        lf->setMaxDetail(maxdetail);
         fileMsgHandler = lf->beginLogging();
+#else
+        Owned<ILogMsgFilter> filter = getCategoryLogMsgFilter(MSGAUD_all, MSGCLS_all, maxdetail);
+        queryLogMsgManager()->changeMonitorFilter(queryStderrLogMsgHandler(), filter);
```

Comment:
When createComponentLogFileCreator is used, it automatically picks up some defaults, e.g. "logfields" (currently from environment.conf).

It would be good to preserve the ability to customize.
In container world you could instead pick up properties that are relevant to logging from the component configuration.

And get with something like:
```
queryComponentConfig().queryProp("log/logfields")
```

the logging level could also be there (may want to default to values from global: section)

Suspect this is a common enough pattern to put these lines in helper function that does it all this, i.e. so each component normally just calls e.g. startComponentLogging();

Reply:
@jakesmith I agree, and I like the idea. I didn't find queryComponentConfig in our code base. How are the other components accessing configuration properties?

Reply:
@rpastrana - queryComponentConfig is in jptree.hpp.

As long as components have used loadConfiguration, they can/do use queryComponentConfig.
All components should call loadConfiguration, I think all the ones we have in containers do.